bcda/auth/client/ssas.go

```diff
@@ -378,3 +378,33 @@ func (c *SSASClient) setAuthHeader(req *http.Request) error {
 	req.Header.Add("Accept", "application/json")
 	return nil
 }
+
+// GetVersion Gets the version of the SSAS client
+func (c *SSASClient) GetVersion() (string, error) {
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s/_version", c.baseURL), nil)
+	if err != nil {
+		return "", err
+	}
+
+	if err := c.setAuthHeader(req); err != nil {
+		return "", err
+	}
+	resp, err := c.Do(req)
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	ssasLogger.Info(resp.Body)
```

Comment:
Curious about this line, do need to log the response body for these requests at the INFO level? Maybe DEBUG or not logging?

And I had thought we could not read the response body multiple times. I believe that this is working now since we're actually stringifying the io.Reader (and not necessarily the contents).

Here's the log output:

```
{"file":"/Users/martintrang/go/src/github.com/CMSgov/bcda-app/bcda/auth/client/ssas.go:398","func":"github.com/CMSgov/bcda-app/bcda/auth/client.(*SSASClient).GetVersion","level":"info","msg":"\u0026{0x1142570 0xc000394040 0x14970e0}","time":"2020-08-18T09:24:42-06:00"}
```